engine/core/src/main/java/org/datacleaner/job/builder/AnalysisJobBuilder.java

```diff
@@ -1368,4 +1368,17 @@ public AnalysisJobBuilder getRootJobBuilder() {
         }
         return consumedOutputDataStreamJobBuilders;
     }
+
+    /**
+     * Checks if a job including its child jobs are correctly configured.
+     **/
+    public boolean isConsumedOutDataStreamsJobBuilderConfigured(final boolean throwException) {
+        final List<AnalysisJobBuilder> consumedOutputDataStreamsJobBuilders = getConsumedOutputDataStreamsJobBuilders();
+        boolean isConfigurated = true;
```

Comment:
should probably be "isConfigured" but actually I think you can avoid it completely and just return false the moment you find something that is not configured, or return true otherwise.